[0m[[0m[31merror[0m] [0m[0mjava.net.BindException: Address already in use[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind(Net.java:459)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.Net.bind(Net.java:448)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/sun.nio.ch.AsynchronousServerSocketChannelImpl.bind(AsynchronousServerSocketChannelImpl.java:164)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.nio.channels.AsynchronousServerSocketChannel.bind(AsynchronousServerSocketChannel.java:198)[0m
[0m[[0m[31merror[0m] [0m[0m	at fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.$anonfun$serverResource$3(SocketGroupPlatform.scala:91)[0m
[0m[[0m[31merror[0m] [0m[0m	at delay @ fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.$anonfun$serverResource$2(SocketGroupPlatform.scala:85)[0m
[0m[[0m[31merror[0m] [0m[0m	at traverse @ fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.serverResource(SocketGroupPlatform.scala:84)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.serverResource(SocketGroupPlatform.scala:84)[0m
[0m[[0m[31merror[0m] [0m[0m	at make @ fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.serverResource(SocketGroupPlatform.scala:140)[0m
[0m[[0m[31merror[0m] [0m[0m	at make @ fs2.io.net.SocketGroupCompanionPlatform$AsyncSocketGroup.serverResource(SocketGroupPlatform.scala:140)[0m
[0m[[0m[31merror[0m] [0m[0m	at uncancelable @ fs2.Pull$.acquireCancelable(Pull.scala:389)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.Compiler$Target.flatMap(Compiler.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m	at handleErrorWith @ fs2.Compiler$Target.handleErrorWith(Compiler.scala:160)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.Compiler$Target.flatMap(Compiler.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m	at uncancelable @ fs2.Compiler$Target.uncancelable(Compiler.scala:164)[0m
[0m[[0m[31merror[0m] [0m[0m	at unique @ org.typelevel.vault.Key$.newKey(Key.scala:112)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.Compiler$Target.flatMap(Compiler.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m	at of @ org.http4s.ember.server.internal.Shutdown$.$anonfun$timedShutdown$2(Shutdown.scala:52)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.Compiler$Target.flatMap(Compiler.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m	at flatMap @ fs2.Compiler$Target.flatMap(Compiler.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) java.net.BindException: Address already in use[0m
